### DISCUSSION TITLE
Failed to load plugin 'jsx-a11y' declared in '.eslintrc.json » eslint-config-next/core-web-vitals

### DISCUSSION THREAD

**Discussion by conchui** [vercel/next.js#58875] [Help] [answered]
### Summary

Encountered an error during yarn build.

Node: 18.17.0
NextJS: 14.0.3
eslint-config-next: 14.0.3

![image](https://github.com/vercel/next.js/assets/8112975/85c659cc-abef-4a9e-aa95-df7d970bfb01)

### Additional information

```js
.eslintrc.json


{
  "plugins": ["@typescript-eslint"],
  "extends": [
    "plugin:@typescript-eslint/recommended",
    "prettier",
    "next/core-web-vitals"
  ],
  "rules": {
    "@typescript-eslint/no-unused-vars": [
      2,
      {
        "argsIgnorePattern": "^_",
        "varsIgnorePattern": "^_",
        "caughtErrorsIgnorePattern": "^_"
      }
    ],
    "@typescript-eslint/no-explicit-any": 0,
    "@typescript-eslint/ban-ts-comment": 0
  }
}
```

**Comment by nqhd3v** [comment]
From my side, I just need to remove folder `node_modules` & `.next`. Re-run `npm install` & `npm run build`.

It works for my side!

![image](https://github.com/vercel/next.js/assets/106160187/2a6c1557-a0dc-4479-9a66-904c29ca8081)

**Comment by alexreardon** [comment]
For anybody else who stumbles on to this, I also needed to run `yarn upgrade` to upgrade all packages to be the highest available versions that satisfied my `package.json`

**Reply by abannachGrafana** [reply]
Depending on your version of `yarn` you may need to run `yarn upgrade-interactive`. Even after doing all that I was still getting hit with the error. I'm on `yarn@4.1.1` I went full nuclear and deleted `node_modules`, `.yarn/cache` and my `yarn.lock`. Not sure exactly what fixed it but I'm guess that the `.yarn/cache` had a corrupt (?) copy of the plugin and kept using it.

In either case, thanks for the bread crumb! It definitely got me across the finish line.

**Reply by JayPe69** [reply]
I tried to only remove node_modules and .yarn/cache ==> No luck
But when removing both folders and the yarn.lock, then it worked.

I also use yarn 4.1.1 (just upgraded to this last version), so maybe there is a relationship between upgrading to yarn 4.1.1 and the error above.

**Reply by philheb** [reply, marked as answer]
removing yarn.lock work for me.

**Comment by aramikuto** [comment]
I was able to fix the error by manually bumping **eslint-plugin-jsx-a11y**. I removed the snippet below from **yarn.lock**:

```text
"eslint-plugin-jsx-a11y@npm:^6.10.0":
  version: 6.10.1
  resolution: "eslint-plugin-jsx-a11y@npm:6.10.1"
  dependencies:
    aria-query: "npm:^5.3.2"
    array-includes: "npm:^3.1.8"
    array.prototype.flatmap: "npm:^1.3.2"
    ast-types-flow: "npm:^0.0.8"
    axe-core: "npm:^4.10.0"
    axobject-query: "npm:^4.1.0"
    damerau-levenshtein: "npm:^1.0.8"
    emoji-regex: "npm:^9.2.2"
    es-iterator-helpers: "npm:^1.1.0"
    hasown: "npm:^2.0.2"
    jsx-ast-utils: "npm:^3.3.5"
    language-tags: "npm:^1.0.9"
    minimatch: "npm:^3.1.2"
    object.fromentries: "npm:^2.0.8"
    safe-regex-test: "npm:^1.0.3"
    string.prototype.includes: "npm:^2.0.1"
  peerDependencies:
    eslint: ^3 || ^4 || ^5 || ^6 || ^7 || ^8 || ^9
  checksum: 10c0/25bf28e3db4f6789c5d4f9300fc6fc54faca19ecc537d0f46e9c873f80ed37103a033e1f716f608fab5f5813dd38af65a9a6ae2e29dd079763ce539ebecf998e
  languageName: node
  linkType: hard
```

After running `yarn install`, **eslint-plugin-jsx-a11y** updated to version 6.10.2, and the error was gone
